index.d.ts

```diff
@@ -153,6 +153,117 @@ interface Request {
   }
 }
 
+interface ContentOptions {
+  /**
+   * Controls the way the HTMLRewriter treats inserted content.
+   *
+   * - true: Raw HTML
+   * - false: Text and any HTML will be escaped
+   */
+  html: boolean
+}
+
+interface Element {
+  /**
+   *  The namespace URI of the element according to Infra Spec
+   * (https://infra.spec.whatwg.org/#namespaces).
+   */
+  namespaceURI: string
+  /**
+   * e.g. "div"
+   */
+  tagName: string
+  /**
+   * Read-Only - key/value pairs of attributes.
+   */
+  readonly attributes: Iterator<{ name: string; value: string }>
+  /**
+   * Indicates whether the element was removed/replaced in a previous handler
+   */
+  removed: boolean
+
+  getAttribute(name: string): string | null
```

Comment:
It would be nice to add descriptions for all these methods. You can just copy-paste them from https://developers.cloudflare.com/workers/reference/apis/html-rewriter